index.html

```diff
@@ -36,6 +36,71 @@
       </nav>
     </div>
   </header>
+  <div class = "section1"> 
+    <div class = "trend"><img src = "assets/rediumpics/trending.png" class="trendingLogo" alt = "trending logo"> <span class = "trending">TRENDING ON REDIUM</span></div>
+    </div>
+    <div class = "tabletSection1">
+      <div class= "section2">
+        <div class = "numbering">01</div>
+        <div class = "content1">
+          <img src = "../readium/assets/rediumpics/rockstar.png" alt = "a user's image" class = "profileImage">
```

Comment:
```suggestion
          <img src = "./assets/rediumpics/rockstar.png" alt = "a user's image" class = "profileImage">
```
```suggestion
          <img src = "./assets/rediumpics/rockstar.png" alt = "a user's image" class = "profileImage">
```

Reply:
The current `../readium/xxx` is redundant. Summarily, it tells the path lookup to go out of the current folder(`../`) then come back in to the current folder (`readium`).  Since we already in the current folder, we can just have it look up the target file directly (`./assets/xxx`).

This also applies to other `img` tags below

Reply:
The current `../readium/xxx` is redundant. Summarily, it tells the path lookup to go out of the current folder(`../`) then come back in to the current folder (`readium`).  Since we already in the current folder, we can just have it look up the target file directly (`./assets/xxx`).

This also applies to other `img` tags below